packages/host/app/resources/room.ts

```diff
@@ -51,6 +53,7 @@ interface SkillId {
   skillCardId: string;
   skillEventId: string;
   isActive: boolean;
+  loadingPromise?: Promise<void>;
```

Comment:
@ef4 was counseling against resources exposing promises -- let's talk this through.